### ISSUE TITLE
Issue with training in CPU

### ISSUE THREAD

**Issue by Haruo195** [harvardnlp/annotated-transformer#48]
Training on CPU with:
torch==1.2.0+cpu torchvision==0.4.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
python 3.5

I have changed the current code as it was throwing warning and error with torch 1.+ as follows:

```
torch.arange(0, max_len) to torch.arange(0.0, max_len) 
torch.exp(torch.arange(0, d_model, 2) to torch.exp(torch.arange(0.0, d_model, 2)
```

`nn.KLDivLoss(size_average=False) to nn.KLDivLoss(reduction='sum')`

`nn.init.xavier_uniform(p) to nn.init.xavier_uniform_(p)`

`Variable(torch.LongTensor([1]))).data[0] to Variable(torch.LongTensor([1])))`

**SimpleLossCompute**
`loss.data[0] to loss`

Also I deleted the In[37] and In[43] as I am training with CPU only.

Now, when I train it, it's showing the following error:

```
Epoch Step: 1 Loss: 3.141923 Tokens per Sec: 540.000000
Floating point exception (core dumped)
```
